frontend-instructor/src/components/coursePage/CoursePage.jsx

```
@@ -0,0 +1,326 @@
+            <button
+              onClick={() => {
+                setShowPublishModal(true);
+              }}
+              className='bg-green-500 hover:bg-green-600 text-white font-medium px-4 py-2 rounded-lg transition shadow-sm flex items-center'
+            >
+              <svg
+                xmlns='http://www.w3.org/2000/svg'
+                className='h-5 w-5 mr-2'
+                viewBox='0 0 20 20'
+                fill='currentColor'
+              >
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)